doc/make.bat

```diff
@@ -10,7 +10,7 @@ if "%SPHINXBUILD%" == "" (
 set SOURCEDIR=.
 set BUILDDIR=build
 set SPHINXPROJ=matplotlib
-set SPHINXOPTS=-W
+set SPHINXOPTS=
```

Comment:
needs to come back

Reply:
Sure. I just need this for testing locally.

Reply:
You can do `make html "SPHINXOPTS="` to override the `-W` flag locally I think.